doc/source/devel/core_developer.rst

```diff
@@ -0,0 +1,166 @@
+.. _core_dev:
+
+Core Developer Guide
+====================
+
+As a core developer, you should continue making pull requests
+in accordance with the :ref:`chap_devguide`.
+You are responsible for shepherding other contributors through the review process.
+You also have the ability to merge or approve other contributors' pull requests.
+
+Reviewing
+---------
+
+How to Conduct A Good Review
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+*Always* be kind to contributors. Nearly all of Nibabel is
+volunteer work, for which we are tremendously grateful. Provide
+constructive criticism on ideas and implementations, and remind
+yourself of how it felt when your own work was being evaluated as a
+novice.
+
+Nibabel strongly values mentorship in code review.  New users
+often need more handholding, having little to no git
+experience. Repeat yourself liberally, and, if you don’t recognize a
+contributor, point them to our development guide, or other GitHub
+workflow tutorials around the web. Do not assume that they know how
+GitHub works (e.g., many don't realize that adding a commit
+automatically updates a pull request). Gentle, polite, kind
+encouragement can make the difference between a new core developer and
+an abandoned pull request.
+
+When reviewing, focus on the following:
+
+1. **API:** The API is what users see when they first use
+   Nibabel. APIs are difficult to change once released, so
+   should be simple, consistent with other parts of the library, and
+   should avoid side-effects such as changing global state or modifying
+   input variables.
+
+2. **Documentation:** Any new feature should have a tutorial
+   example that not only illustrates but explains it.
+
+3. **The algorithm:** You should understand the code being modified or
+   added before approving it.  (See `Merge Only Changes You
+   Understand`_ below.) Implementations should do what they claim,
+   and be simple, readable, and efficient.
+
+4. **Tests:** All contributions to the library *must* be tested, and
+   each added line of code should be covered by at least one test. Good
+   tests not only execute the code, but explores corner cases.  It is tempting
+   not to review tests, but please do so.
+
+Other changes may be *nitpicky*: spelling mistakes, formatting,
+etc. Do not ask contributors to make these changes, and instead
+make the changes by `pushing to their branch
+<https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/committing-changes-to-a-pull-request-branch-created-from-a-fork>`__,
+or using GitHub’s `suggestion
+<https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request>`__
+`feature
+<https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request>`__.
+(The latter is preferred because it gives the contributor a choice in
+whether to accept the changes.)
+
+Our default merge policy is to squash all PR commits into a single
```

Comment:
I think we aren't doing that at the moment.  Do we want to?   I personally prefer leaving the commits as they are.

Reply:
Ah, right. Meant to add a discussion point on this. Yeah, I'm fine sticking with merge commits, but it means we need to rewrite or remove this whole paragraph.

Reply:
I think removing the paragraph is OK if that's OK with you (I mean - sticking to default previous merge policy).